content/docs/getting-started/futures.md

```diff
@@ -205,6 +205,8 @@ use std::time::Duration;
 
 use futures::Future;
 use futures_cpupool::CpuPool;
+// There's also a tokio_core::reactor::Timeout, but it needs an event loop
+// to run on.
```

Comment:
OK, how about the fixup commit? (I'd squash it once we agree if that's OK)

I'm a bit worried people would start using the `tokio-timer` crate for their general timeouts as they see in the example and I'm afraid starting another thread is an overkill most of the time, so I want to point the existence of the core timeouts out.